src/ship/PlayerShipController.cpp

```diff
@@ -688,6 +686,8 @@ void PlayerShipController::PollControls(const float timeStep, const bool force_r
 	if (InputBindings.killRot->IsActive()) SetFlightControlState(CONTROL_FIXHEADING_KILLROT);
 
 	outParams.desiredAngular = wantAngVel;
+	outParams.desireAngVel = force_rotation_damping || m_rotationDamping || wantAngVel.Length() >= 0.001;
```

Comment:
In fact `desireAngVel` is a runtime type specifier for `desiredAngular`, it seems worth explaining here why we can so easily convert a speed request to a thrust request. Also, the question is, is the desired behavior obtained? Should we limit the rotation speed if we have rotation damping disabled?

Reply:
I've always assumed that the maximum rotational speed with dampening on is based on ship and pilot structural integrity, so it would make sense to have a limit with dampening off as well. If people really are opposed to it, there could be some sort of "safeties off" mode one could apply. There should be some kind of damage to the ship done for rotating beyond tolerances with safeties off, in that case!

Reply:
The limit for rotation speed is not disabled with rotation damping off. In fact the change is only effective if the player is not pressing any rotation button or in general if rotation requested is close to zero. I tested with rotation damping turned off and I was unable to spin the ship faster then with damping off.
When ship hits something with rotation damping of it could spin faster then limit though, but that should be expected.
From my point of view the desired behavior is obtained.

Reply:
> it seems worth explaining here why we can so easily convert a speed request to a thrust request

I'm not sure I understand what you expect. The desireAngVel is set to false only if we have 0 requested  rotation from inputs this translates to 0 thruster request which is correct in case of rotation damping off.
Do you want me to add a comment about that?

Reply:
@azieba Yeah, something like that, I just personally think it's not quite obvious from the code, and variable names are not conducive to easy understanding.